website/docs/contributing/ADRs.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Architectural Decision Records
+---
+
+## Introduction {#introduction}
+
+Architectural decision records are a record of design decisions we have made in the past because we belived they would help our code quality over time. Any ADR can be challenged, but two conditions must be met to change an ADR: (1) The proposed solution must provide a tangible benefit in terms of code quality, and (2) the proposed solutions benefit must be worth retroactively refitted to the entire codebase. One such example is the decision to re-write unleash to typescript.
+
+## Crosscutting ADRs
+
+* [Domain language](./general/ADR/domain-language.md)
+
+## Backend ADRs
```

Comment:
Now that this goes into the "official" docs, I think we should align on how we spell "back end" and "front end". I've been going off Chris Coyier's short [post on whether it's "front-end" or front end"](https://css-tricks.com/poll-results-front-end-front-end/) for a bit, which says that:
- as a noun, it's two words. For instance: "I work on the front end"
- as a compound adjective, it's with a hyphen. E.g: "I'm a front-end engineer"

But I would actually be very interested to hear @JarrodSC's thoughts on this.

```suggestion
## Back-end ADRs
```